tests/spread/extensions/gnome-3-28/task.yaml

```diff
@@ -12,13 +12,17 @@ systems:
   - ubuntu-18.04-armhf
 
 environment:
-  SNAP_DIR: ../snaps/glib-hello
+  SNAP_DIR: ../snaps/gtk3-hello
 
 prepare: |
   #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
   . "$TOOLS_DIR/snapcraft-yaml.sh"
   set_base "$SNAP_DIR/snap/snapcraft.yaml"
 
+  # Remove content snaps to be installed by test.
```

Comment:
This really shouldn't be needed, I suspect this returning 0 may even be a bug and will hit us in the future.